src/app.rs

```diff
@@ -422,6 +429,7 @@ impl<S, E, X> ApplicationBuilder<S, E, X> {
     /// game.run();
     /// ~~~
 
+    // TODO: Unused lifetime. Is it required?
```

Comment:
I'd venture a "no"

Reply:
No, it's a remnant of old code that's been refactored atleast three times :)